packages/ag-charts-website/src/features/api-documentation/utils/getTypeLinks.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * These are used to create links from types to relevant documentation.
+ */
+const TYPE_LINKS = {
+    AgChartTheme: '/integrated-charts-customisation/#custom-chart-themes',
+    AgChartThemeName: '/integrated-charts-customisation/#provided-themes',
+    AgChartThemeOverrides: '/charts-api-themes/#reference-theme.palette',
+    AgChartThemePalette: '/charts-api-themes/#reference-theme.overrides',
+    AgCartesianChartOptions: '/integrated-charts-customisation/#overriding-existing-themes',
+    AgCartesianAxesTheme: '/integrated-charts-customisation/#overriding-existing-themes',
+    AgCartesianSeriesTheme: '/integrated-charts-customisation/#overriding-existing-themes',
+    AgBarSeriesOptions: '/integrated-charts-customisation/#example-barcolumn-chart-overrides',
+    AgLineSeriesOptions: '/integrated-charts-customisation/#example-line-chart-overrides',
+    AgAreaSeriesOptions: '/integrated-charts-customisation/#example-area-chart-overrides',
+    AgScatterSeriesOptions: '/integrated-charts-customisation/#example-scatter-chart-overrides',
+    AgHistogramSeriesOptions: '/integrated-charts-customisation/#example-histogram-chart-overrides',
+    AgPolarChartOptions: '/integrated-charts-customisation/#example-piedoughnut-chart-overrides',
+    AgPolarAxesTheme: '/integrated-charts-customisation/#example-piedoughnut-chart-overrides',
+    AgPolarSeriesTheme: '/integrated-charts-customisation/#example-piedoughnut-chart-overrides',
+    AgPieSeriesOptions: '/integrated-charts-customisation/#example-piedoughnut-chart-overrides',
+    Blob: 'https://developer.mozilla.org/en-US/docs/Web/API/Blob',
+    CellPosition: '/keyboard-navigation/#cellposition',
+    CellRange: '/range-selection/#range-selection-api',
+    ICellRendererParams: '/component-cell-renderer/#reference-ICellRendererParams',
+    ICellEditorParams: '/component-cell-editor/#reference-ICellEditorParams',
+    ChartModel: '/integrated-charts-api-save-restore-charts/#api-reference',
```

Comment:
Some of these types are Grid types - maybe we could drop them, or break them into a distinct definition to make it clearer?

Reply:
I've got a todo to come back to this. We'll need to see how it's being used in Charts (or whether there is a better way)